### PR TITLE
Added fields to address, news, material models; modified orders model

### PR DESCRIPTION
**Pull request by YukiAmeka**

### CODE REVIEW

File: modules/material/material.graphql.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const materialType = `
     _id: ID!
     name: [Language]
     description: [Language]
+    purpose: String
```
Comment (Alan18081) [inline modules/material/material.graphql.js:6]: What is `purpose`?

File: modules/common/Address.js [hunk 2]
```diff
@@ -2,7 +2,9 @@ const mongoose = require('mongoose');
 
 const addressSchema = new mongoose.Schema({
   country: String,
+  region: String,
```
Comment (Alan18081) [inline modules/common/Address.js:5]: Where are we going to use `Address`?

File: modules/material/material.graphql.js (hunk 1 above)
Comment (YukiAmeka) [reply modules/material/material.graphql.js:6]: A field to differentiate between "main" and "bottom" materials for the material page representation. We discussed it under the `type` name, but `type`, though not reserved, has a use in mongoose, so I'd rather avoid it

File: modules/common/Address.js (hunk 2 above)
Comment (YukiAmeka) [reply modules/common/Address.js:5]: In users

File: modules/material/material.graphql.js (hunk 1 above)
Comment (Alan18081) [reply modules/material/material.graphql.js:6]: I think, it is better to rename it to the `materialType`, because it reflects type

Comment (YukiAmeka) [reply modules/material/material.graphql.js:6]: No, it doesn't. It reflects the `purpose` for which the material is used in this context, it has nothing to do with the material's intrinsic characteristics

**Pull request merged**